File: CompSci 1/Unit 18, Arrays/Reverse/Reverse.java
import java.util.Scanner;

public class Reverse
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        int[] nums = new int[20];

        System.out.println("Enter 20 integers: ");
        for(int i = 0; i < 20; i++)
        {
            System.out.print("Num " + (i + 1) + ": ");
            nums[i] = input.nextInt();
        }

        for(int i : nums)
        {
            System.out.print(i + " ");
        }

        System.out.println();
        System.out.println();

        for(int i = nums.length - 1; i >= 0; i--)
        {
            System.out.print(nums[i] + " ");
        }
    }
}

File: CompSci 1/Unit 18, Arrays/Reverse/Reverse.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val input = Scanner(System.`in`)
    val nums = IntArray(20)

    println("Enter 20 integers: ")
    for (i in 0..19) {
        print("Num " + (i + 1) + ": ")
        nums[i] = input.nextInt()
    }

    for (i in nums) {
        print(i.toString() + " ")
    }

    println()
    println()

    for (i in nums.indices.reversed()) {
        print(nums[i].toString() + " ")
    }
}